addons/binding/org.openhab.binding.mihome/ESH-INF/thing/sensor_wleak_aq1.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="water">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge" />
+        </supported-bridge-type-refs>
+        <label>Xiaomi Water Leak Sensor</label>
+        <channels>
+            <channel id="leak" typeId="leak"></channel>
+        </channels>
```

Comment:
are you sure there are no battery channels?
            <channel id="batteryLevel" typeId="system.battery-level" />
            <channel id="lowBattery" typeId="system.low-battery" />